docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst

```diff
@@ -0,0 +1,52 @@
+.. _pro-check-list-cves:
+
+Given a list of CVEs, how to know if you are affected by them?
```

Comment:
How to check a system against a list of known CVEs

Reply:
Doesn't have to be this specific wording (this is just a suggestion) but the page title is the main thing that helps in search results/discoverability so we want to try and get it to match as specifically as possible to the contents of the page. It's also the first thing a reader will skim to judge if the page is likely what they're looking for. Something like "How to know if a system is affected by a list of CVEs" could also possibly work.